tests/main/ubuntu-core-netplan-apply/task.yaml

```diff
@@ -0,0 +1,55 @@
+summary: Ensure that netplan apply works on Ubuntu Core with network-setup-control
+
+details: |
+    Netplan apply is used to apply network configuration to the system
+
+environment:
+    NETPLAN: io.netplan.Netplan
+
+# only run on ubuntu core systems
+systems:
+    - ubuntu-core-*
+
+prepare: |
+    snap install test-snapd-netplan-apply --edge
```

Comment:
Is this snap something that we must get from the store for correctness or just for convenience?

Reply:
It has to be built with snapcraft, so we could try to build it in the test but this seems a bit heavy for a test job to need to install snapcraft, etc. especially since we are now on core for this test and this snap is re-used from the fake-netplan test so it would need to be built there too.